en/sql/tuning.rst

```diff
@@ -3829,6 +3829,89 @@ The join with the *right_tbl* table was eliminated.
                 9           60          600  'Left- 806'                     6
                10           60          600  'Left- 906'                     6
 
+.. _pred-push:
+
+Predicate Push
+-----------------------
+
+**Predicate Push** is an optimization that pushes the predicate of the main query into the view.
+
+By applying **Predicate Push** to the subquery, the amount of data queried in the main query is reduced.
+
+For instance, when performing a join using the join condition *a.code = r.athlete_code* as in the query below, if the external predicate can be pushed into the query block, the amount of data to be joined can be reduced.
+
+.. code-block:: sql
+
+        SELECT a.name, r.score 
+        FROM (SELECT name, nation_code, code, count(*) cnt FROM athlete GROUP BY name, nation_code) a, record r
+        WHERE a.code = r.athlete_code
+        AND a.nation_code = 'KOR';
+
+There are no predicates in the inline-view of the query above. If the query transformation doesn't work, 
+the *athlete* table would have been fully scanned and joined before filtering with the condition *a.nation_code = 'KOR'*.
+
+However, with **Predicate Push**, if the query is transformed as follows, the optimization ensures that only fewer amounts of data are queried.
```

Comment:
How about below?

However, if the query is transformed as follows by using **Predicate Push**, it can be optimized to reduce the amount of data being queried.